app/mixins/check-login-mixin.js

```diff
@@ -14,6 +15,11 @@ export default Ember.Mixin.create({
             }
         }).then(function(loggedIn) {
             if (loggedIn.data === 'false') {
+                if (window.location.href !== "http://localhost:4200/") {
+                    document.cookie = "redirectURL=" + window.location.href;
+                } else {
+                    document.cookie = "redirectURL=http://localhost:4200/conference/new/";
```

Comment:
We really can't have a hardcoded value here. Please put in the config file and then import.